core/templates/dev/head/services/speech-synthesis-chunker.service.spec.ts

```diff
@@ -32,87 +31,199 @@ describe('Speech Synthesis Chunker Service', () => {
     speechSynthesisChunkerService = TestBed.get(SpeechSynthesisChunkerService);
   });
 
-  it('Should properly convert subtraction in LaTeX to speakable text',
-    () => {
-      var latex1 = '5-3';
-      var latex2 = 'i-j';
-      var speakableLatex1 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex1);
-      var speakableLatex2 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex2);
-      expect(speakableLatex1).toEqual('5 minus 3');
-      expect(speakableLatex2).toEqual('i minus j');
-    }
-  );
-
-  it('Should properly convert fractions in LaTeX to speakable text',
-    () => {
-      var latex1 = '\\\\frac{2}{3}';
-      var latex2 = '\\\\frac{abc}{xyz}';
-      var latex3 = '\\\\frac{3n}{5}';
-      var latex4 = '\\\\frac{ijk}{5xy}';
-      var speakableLatex1 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex1);
-      var speakableLatex2 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex2);
-      var speakableLatex3 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex3);
-      var speakableLatex4 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex4);
-      expect(speakableLatex1).toEqual('2/3');
-      expect(speakableLatex2).toEqual('a b c over x y z');
-      expect(speakableLatex3).toEqual('3n over 5');
-      expect(speakableLatex4).toEqual('i j k over 5x y');
-    }
-  );
-
-  it('Should properly convert square roots in LaTeX to speakable text',
-    () => {
-      var latex1 = '\\\\sqrt{3}';
-      var latex2 = '\\\\sqrt{xy}';
-      var speakableLatex1 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex1);
-      var speakableLatex2 =
-        speechSynthesisChunkerService.formatLatexToSpeakableText(latex2);
-      expect(speakableLatex1).toEqual('the square root of 3');
-      expect(speakableLatex2).toEqual('the square root of x y');
-    }
-  );
-
-  it('Should properly convert exponents in LaTeX to speakable text',
-    () => {
-      var latex1 = 'x^2';
-      var latex2 = '42^4';
-      var latex3 = 'x^62';
-      var latex4 = '3n^4x';
+  describe('formatLatexToSpeakableText', () => {
+    it('should properly convert subtraction in LaTeX to speakable text',
+      () => {
+        var latex1 = '5-3';
```

Comment:
Add a space before and after `-`(just a minor nit, here and elsewhere)

Reply:
Done.